<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>presenter</class>
 <widget class="QMainWindow" name="presenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>presenter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="Pawn" name="firstPawn" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="Pawn" name="widget_2" native="true"/>
    </item>
    <item row="0" column="2">
     <widget class="Pawn" name="widget_3" native="true"/>
    </item>
    <item row="0" column="3">
     <widget class="Pawn" name="widget_4" native="true"/>
    </item>
    <item row="0" column="4">
     <widget class="Pawn" name="widget_5" native="true"/>
    </item>
    <item row="0" column="5">
     <widget class="Pawn" name="widget_6" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="Pawn" name="secondPawn" native="true"/>
    </item>
    <item row="1" column="1">
     <widget class="Pawn" name="widget_13" native="true"/>
    </item>
    <item row="1" column="2">
     <widget class="Pawn" name="widget_8" native="true"/>
    </item>
    <item row="1" column="3">
     <widget class="Pawn" name="widget_14" native="true"/>
    </item>
    <item row="1" column="4">
     <widget class="Pawn" name="widget_11" native="true"/>
    </item>
    <item row="1" column="5">
     <widget class="Pawn" name="widget_12" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="Pawn" name="widget_17" native="true"/>
    </item>
    <item row="2" column="1">
     <widget class="Pawn" name="widget_20" native="true"/>
    </item>
    <item row="2" column="2">
     <widget class="Pawn" name="widget_15" native="true"/>
    </item>
    <item row="2" column="3">
     <widget class="Pawn" name="widget_21" native="true"/>
    </item>
    <item row="2" column="4">
     <widget class="Pawn" name="widget_18" native="true"/>
    </item>
    <item row="2" column="5">
     <widget class="Pawn" name="widget_19" native="true"/>
    </item>
    <item row="3" column="0">
     <widget class="Pawn" name="widget_24" native="true"/>
    </item>
    <item row="3" column="1">
     <widget class="Pawn" name="widget_27" native="true"/>
    </item>
    <item row="3" column="2">
     <widget class="Pawn" name="widget_22" native="true"/>
    </item>
    <item row="3" column="3">
     <widget class="Pawn" name="widget_28" native="true"/>
    </item>
    <item row="3" column="4">
     <widget class="Pawn" name="widget_25" native="true"/>
    </item>
    <item row="3" column="5">
     <widget class="Pawn" name="widget_26" native="true"/>
    </item>
    <item row="4" column="0">
     <widget class="Pawn" name="widget_31" native="true"/>
    </item>
    <item row="4" column="1">
     <widget class="Pawn" name="widget_34" native="true"/>
    </item>
    <item row="4" column="2">
     <widget class="Pawn" name="widget_29" native="true"/>
    </item>
    <item row="4" column="3">
     <widget class="Pawn" name="widget_35" native="true"/>
    </item>
    <item row="4" column="4">
     <widget class="Pawn" name="widget_32" native="true"/>
    </item>
    <item row="4" column="5">
     <widget class="Pawn" name="widget_33" native="true"/>
    </item>
    <item row="5" column="0">
     <widget class="Pawn" name="widget_38" native="true"/>
    </item>
    <item row="5" column="1">
     <widget class="Pawn" name="widget_41" native="true"/>
    </item>
    <item row="5" column="2">
     <widget class="Pawn" name="widget_36" native="true"/>
    </item>
    <item row="5" column="3">
     <widget class="Pawn" name="widget_42" native="true"/>
    </item>
    <item row="5" column="4">
     <widget class="Pawn" name="widget_39" native="true"/>
    </item>
    <item row="5" column="5">
     <widget class="Pawn" name="widget_40" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save your current grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open a grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create a new grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Pawn</class>
   <extends>QWidget</extends>
   <header>Pawn.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
